06-aks-cluster.md

```diff
@@ -4,134 +4,133 @@ Now that the [cluster prequisites and shared Azure service instances are provisi
+
+        > :bulb:
+        >
+        > - Use the env var value of `AKS_INGRESS_CONTROLLER_CERTIFICATE_BASE64`
+        > - Ideally fetching this secret from a platform-managed secret store such as [Azure Key Vault](https://github.com/marketplace/actions/azure-key-vault-get-secrets)
+
+        ```bash
+        echo $AKS_INGRESS_CONTROLLER_CERTIFICATE_BASE64
+        ```
+
+    1.  Copy the GitHub workflow file into the expected directory
+
+        ```bash
+        mkdir -p .github/workflows
+        cat github-workflow/aks-deploy.yaml > .github/workflows/aks-deploy.yaml
+        ```
+
+    1.  Generate cluster parameter file per region
+
+        ```bash
+        #Region1
+        cat ./azuredeploy.parameters.region.json | \
+        sed "s#<resource-location>#eastus2#g" | \
+        sed "s#<geo-redundancy-location>#centralus#g" | \
+        sed "s#<cluster-spoke-vnet-resource-id>#${RESOURCEID_VNET_BU0001A0042_03}#g" | \
+        sed "s#<tenant-id-with-user-admin-permissions>#${TENANTID_K8SRBAC}#g" | \
+        sed "s#<azure-ad-aks-admin-group-object-id>#${AADOBJECTID_GROUP_CLUSTERADMIN}#g" | \
+        sed "s#<cluster-internal-load-balancer-ip-address>#10.243.4.4#g" | \
+        sed "s#<app-instance-id>#03#g" | \
+        sed "s#<log-analytics-workspace-id>#${LOGANALYTICSWORKSPACEID}#g" | \
+        sed "s#<container-registry-id>#${CONTAINERREGISTRYID}#g" | \
+        sed "s#<acrPrivateDns-zones-id>#${ACRPRIVATEDNSZONESID}#g" | \
+        sed "s#<subdomain-name>#${CLUSTER_SUBDOMAIN_03}#g" \
+        > azuredeploy.parameters.region1.json
+
+        #Region2
+        cat ./azuredeploy.parameters.region.json | \
+        sed "s#<resource-location>#centralus#g" | \
+        sed "s#<geo-redundancy-location>#centralus#g" | \
+        sed "s#<cluster-spoke-vnet-resource-id>#${RESOURCEID_VNET_BU0001A0042_04}#g" | \
+        sed "s#<tenant-id-with-user-admin-permissions>#${TENANTID_K8SRBAC}#g" | \
+        sed "s#<azure-ad-aks-admin-group-object-id>#${AADOBJECTID_GROUP_CLUSTERADMIN}#g" | \
+        sed "s#<cluster-internal-load-balancer-ip-address>#10.244.4.4#g" | \
+        sed "s#<app-instance-id>#04#g" | \
+        sed "s#<log-analytics-workspace-id>#${LOGANALYTICSWORKSPACEID}#g" | \
+        sed "s#<container-registry-id>#${CONTAINERREGISTRYID}#g" | \
+        sed "s#<acrPrivateDns-zones-id>#${ACRPRIVATEDNSZONESID}#g" | \
+        sed "s#<subdomain-name>#${CLUSTER_SUBDOMAIN_04}#g" \
+        > azuredeploy.parameters.region2.json
+        ```
+
+    1.  Push the changes to your forked repo.
+
+        > :book: The DevOps team wants to automate their infrastructure deployments. In this case, they decided to use GitHub Actions. They are going to create a workflow for every AKS cluster instance they have to take care of.
+
+        ```bash
+        git add .github/workflows/aks-deploy.yaml
+        git add  azuredeploy.parameters.region1.json
```

Comment:
please lets try to replace these lines by `git add -u && git commit -m "setup GitHub CD workflow" && git push origin main`

Reply:
done